conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -595,6 +583,13 @@ private void setSeekStart(int seekTarget) {
     public void setSeekEnd() {
         Log.d(TAG, "Sending seek end event");
         convivaVideoAnalytics.reportPlaybackMetric(ConvivaSdkConstants.PLAYBACK.SEEK_ENDED);
+        // Notify of seek buffering complete at this stage.
+        Log.d(TAG, "[Player Event] Update state after buffering");
+        ConvivaSdkConstants.PlayerState state = ConvivaSdkConstants.PlayerState.PAUSED;
```

Comment:
Any specific reason for keeping default state as PAUSED? In the previous logic as well as the logic in StallEnded event handling, the default state was PLAYING.

Reply:
I flipped the logic because I noticed that isPaused() will report false if the video wasn't started prior to seeking.

Reply:
ok..good catch:)